### PR TITLE
Fix helm hub release

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Review by johnma14 — APPROVED** [review]

File: release/publish.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ VERSION="$(cat "${WD}/trigger-publish")"
 SOURCE_GCS_BUCKET=${SOURCE_GCS_BUCKET:-istio-prerelease/prerelease}
 GCS_BUCKET=${GCS_BUCKET:-istio-release/releases}
 HELM_BUCKET=${HELM_BUCKET:-istio-release/charts}
-HELM_BUCKET_RELEASE=${HELM_BUCKET:-gcr.io/istio-release/charts}
+HELM_HUB_RELEASE=${HELM_HUB_RELEASE:-gcr.io/istio-release/charts}
```
Comment (ericvn) [inline release/publish.sh:43]: Does line 71 also need a change with this?

Comment (howardjohn) [reply release/publish.sh:43]: yes ty

**Review by howardjohn — COMMENTED** [review]
fyi this is a regression from https://github.com/istio/release-builder/pull/1254 and partially reverts it, @jtischer-acasi you may need to update your env var name if you are setting this

File: release/publish.sh (hunk 1 above)
Comment (ericvn) [reply release/publish.sh:43]: Thanks John. Seems like this should be good (adding the gcr.io to what was there).

**Pull request merged**